web/src/hooks/viewport.js

```diff
@@ -45,3 +45,36 @@ export function useHeightObserver(ref, offset = 0) {
 
   return height;
 }
+
+export function useWindowSize() {
+  const [windowSize, setWindowSize] = useState({});
+
+  // Resize hook
+  useEffect(() => {
+    const updateSize = () => {
+      setWindowSize({
+        width: window.innerWidth,
+        height: window.innerHeight,
+      });
+    };
+    // Set initial size
+    updateSize();
+    // Add window listeners
+    window.addEventListener('resize', updateSize);
+    return () => {
+      window.removeEventListener('resize', updateSize);
+    };
+  });
+
+  return windowSize;
+}
+
+export function useIsSmallScreen() {
+  const { width } = useWindowSize();
+  return width < 768;
+}
+
+export function useIsMediumUpScreen() {
+  const { width } = useWindowSize();
+  return width >= 768;
+}
```

Comment:
NIce hooks :-)

Reply:
Thanks :wink: And thank you for the review! :pray: